datasets/stsb_mt_sv/dataset_infos.json

```diff
@@ -0,0 +1 @@
+{"plain_text": {"description": "The stsb-mt-sv corpus (version 1.0) is a machine translated Swedish version \nof the original STS-B (http://ixa2.si.ehu.eus/stswiki).\n", "citation": "@article{isbister2020not,\n  title={Why Not Simply Translate? A First Swedish Evaluation Benchmark for Semantic Similarity},\n  author={Isbister, Tim and Sahlgren, Magnus},\n  journal={arXiv preprint arXiv:2009.03116},\n  year={2020}\n}\n", "homepage": "https://github.com/timpal0l/sts-benchmark-swedish", "license": "", "features": {"sentence1": {"dtype": "string", "id": null, "_type": "Value"}, "sentence2": {"dtype": "string", "id": null, "_type": "Value"}, "score": {"dtype": "float32", "id": null, "_type": "Value"}}, "post_processed": null, "supervised_keys": null, "builder_name": "swedish_translated_stsb", "config_name": "plain_text", "version": {"version_str": "1.0.0", "description": "", "major": 1, "minor": 0, "patch": 0}, "splits": {"test": {"name": "test", "num_bytes": 171823, "num_examples": 1379, "dataset_name": "swedish_translated_stsb"}, "validation": {"name": "validation", "num_bytes": 218843, "num_examples": 1500, "dataset_name": "swedish_translated_stsb"}, "train": {"name": "train", "num_bytes": 772847, "num_examples": 5749, "dataset_name": "swedish_translated_stsb"}}, "download_checksums": {"https://raw.githubusercontent.com/timpal0l/sts-benchmark-swedish/master/data/stsb-mt-sv.zip": {"num_bytes": 383047, "checksum": "f7f941887a6fea53efa4379c0a5338ee0ea0b704debabbd2dd8dec9c2ef4891f"}}, "download_size": 383047, "post_processing_size": null, "dataset_size": 1163513, "size_in_bytes": 1546560}}
```

Comment:
this file needs to be regenerated after the class name update

Reply:
Could you regenerate it and open a PR @ timpal0l please ?
sorry we didn't spot it earlier

Reply:
@lhoestq Thanks ---> #1138